From 9445d131fc65a2a86f8b005f3988838fa26fb60c Mon Sep 17 00:00:00 2001
From: Brandon McAnsh <brandonm@matricom.net>
Date: Fri, 18 Apr 2014 13:57:26 -0400
Subject: [PATCH 4/4] DVDFactoryCodec: Amlogic: Force rmvb to ffmpeg

Signed-off-by: Brandon McAnsh <brandonm@matricom.net>
---
 xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp b/xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp
index 912dc20..4945aee 100644
--- a/xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp
+++ b/xbmc/cores/dvdplayer/DVDCodecs/DVDFactoryCodec.cpp
@@ -213,6 +213,10 @@ CDVDVideoCodec* CDVDFactoryCodec::CreateVideoCodec(CDVDStreamInfo &hint, unsigne
           if ( (pCodec = OpenCodec(new CDVDVideoCodecAmlogic(), hint, options)) ) return pCodec;
         }
         break;
+      case AV_CODEC_ID_RV10:
+      case AV_CODEC_ID_RV20:
+      case AV_CODEC_ID_RV40:
+        break;
       default:
         if ( (pCodec = OpenCodec(new CDVDVideoCodecAmlogic(), hint, options)) ) return pCodec;
         break;
-- 
1.7.9.5

